packages/firebase_ai/firebase_ai/example/lib/utils/audio_output.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:typed_data';
+
+import 'package:flutter_soloud/flutter_soloud.dart';
+
+class AudioOutput {
+  AudioSource? stream; // Start playback
+  SoundHandle? handle;
+
+  Future<void> init() async {
+    /// Initialize the player.
```

Comment:
[nit] use `//` for comments that are not member docs.

```suggestion
    // Initialize the player.
```